geoapps/io/MagneticVector/constants.py

```diff
@@ -424,7 +435,18 @@
     },
     "out_group": {"label": "Results group name", "value": "VectorInversion"},
 }
+
 default_ui_json.update(base_default_ui_json)
+# for k, v in inversion_defaults.items():
+#     if isinstance(default_ui_json[k], dict):
+#         key = "value"
+#         if "isValue" in default_ui_json[k].keys():
+#             if default_ui_json[k]["isValue"] == False:
+#                 key = "property"
+#         default_ui_json[k][key] = v
+#     else:
+#         default_ui_json[k] = v
+
```

Comment:
Why commented out here?

Reply:
uncommented. don't know why that got sidelined..